tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -348,3 +348,91 @@ TEST(caffe2, parallelBatchedMatmulRHS) {
   // We don't actually check that the output is correct, because this
   // should be covered in the OperatorTest for MatMul already.
 }
+
+/// Test loading clip op from a Caffe2 model.
+TEST(caffe2, importClip) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename("tests/models/caffe2Models/clip_op_net.pbtxt");
+  std::string NetWeightFilename(
+      "tests/models/caffe2Models/empty_init_net.pbtxt");
+
+  Variable *output;
+  Tensor inputs_0(ElemKind::FloatTy, {5, 5});
+  std::vector<float> testValues = {45.0, 16.0, 59.0, 99.0, 48.0, 12.0, 44.0,
+                                   46.0, 82.0, 28.0, 1.0,  91.0, 18.0, 9.0,
+                                   71.0, 24.0, 37.0, 61.0, 12.0, 81.0, 36.0,
+                                   38.0, 30.0, 84.0, 40.0};
+
+  for (size_t i = 0; i < 5 * 5; i++) {
+    inputs_0.getHandle().raw(i) = testValues[i];
+  }
+
```

Comment:
Since `std::vector<float> testValues` is only used for initialization, it can be removed and write : inputs_0.getHandle<>() = {...};